docs/install/command-line.md

```diff
@@ -69,21 +69,21 @@ The IPFS team manages the [dist.ipfs.io website](https://dist.ipfs.io/) to help
    cd ~
    ipfs --version
 
-   > ipfs version 0.8.0
+   > ipfs version 0.9.0
    ```
 
 ### macOS
 
 1. Download the macOS binary from [`dist.ipfs.io`](https://dist.ipfs.io/#go-ipfs).
 
    ```bash
-   wget https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_darwin-amd64.tar.gz
+   wget https://dist.ipfs.io/go-ipfs/v0.9.0/go-ipfs_v0.9.0_darwin-amd64.tar.gz
    ```
```

Comment:
We may want to add a comment here about how you can grab the ones built for Darwin ARM64 (i.e. the M1 hardware) at https://dist.ipfs.io/go-ipfs/v0.9.0/go-ipfs_v0.9.0_darwin-arm64.tar.gz

Reply:
Good call. I've added a call out to tell users how to install Go-IPFS on M1-based Macs.